### PR TITLE
[WARLOCK S4 DF] Update DF4 GENERATORS FOR WARLOCKS.

### PR DESCRIPTION
**Pull request by Mlklock**
DF4 Warlocks Profiles, some changes may still apply.

Changed Gera for Destro, Aff, Demo

Changed Talent for Destro.

### CODE REVIEW

File: profiles/generators/DF4/DF4_Generate_Warlock.simc [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
+# wrists=vibrant_wildercloth_wristwraps,id=193510,bonus_id=9500/9405/9379/8960/523/8793,ilevel=525,gem_id=192964,crafted_stats=40/49
+# hands=grips_of_the_sinister_savant,id=217211,ilevel=528
+# waist=magathas_spiritual_sash,id=195515,bonus_id=1808,ilevel=528,gem_id=192964,enchant_id=6904
+# legs=leggings_of_the_sinister_savant,id=217213,ilevel=528,enchant_id=6830
+# feet=vibrant_wildercloth_slippers,id=193519,bonus_id=8932/8960/8793,ilevel=525
+# finger1=seal_of_filial_duty,id=195526,bonus_id=4800/4786/1498/523,ilevel=535,gem_id=192964,enchant_id=6550
+# finger2=seal_of_diurnas_chosen,id=195480,bonus_id=1808,ilevel=528,gem_id=192964,enchant_id=6550
+# trinket1=whispering_incarnate_icon,id=194301,bonus_id=4800/4786/1498,ilevel=528
+# trinket2=nymues_unraveling_spindle,id=208615,bonus_id=4800/4786/1520/9576,ilevel=528
+# main_hand=dreambinder_loom_of_the_great_cycle,id=208616,ilevel=528,enchant_id=6655
```
Comment (seanpeters86) [inline profiles/generators/DF4/DF4_Generate_Warlock.simc]: need to remove the comments

Comment (Mlklock) [reply profiles/generators/DF4/DF4_Generate_Warlock.simc]: Sorry about that.

**Comment by Mlklock** [comment]
https://www.raidbots.com/simbot/report/6XCR5R2CLyNAiwdNM1qUn8

https://www.raidbots.com/simbot/report/uxs8sJAdzaiTdHG1C4Jk2B 

https://www.raidbots.com/simbot/report/eLmJS3EGbazYZiTqc8bhFM

Removed the Nerfed Idol Trinket, swapped the gems towards Crit and some gear parts to account for the crit loss.

**Review by Mlklock — COMMENTED** [review]
Some Changes due to Trinket nerf, others are just improvements i found around.

**Pull request merged**